README.asciidoc

```diff
@@ -47,5 +50,42 @@ and we can use VNC to connect to the QEMU instance:
 
 $ vncviewer localhost:91 -ViewOnly -Shared
 
+How to contribute
+-----------------
+
 If you want to contribute to this project, please clone and send
-git-pull requests via https://github.com/os-autoinst/os-autoinst
+git-pull requests via https://github.com/os-autoinst/os-autoinst.
+
+More information on the contribution can be found on
+http://os-autoinst.github.io/openQA/contact/, too.
+
+Issues are tracked on https://progress.opensuse.org/projects/openqav3/ and
+also on https://github.com/os-autoinst/os-autoinst/issues
+
+Rules for commits
+^^^^^^^^^^^^^^^^^
+
+* Every commit is checked by https://travis-ci.org/travis[Travis CI] as soon as
+you create a pull request but you *should* run the os-autoinst tests locally,
+i.e. call
+----
+./autogen.sh
+./configure
+make
+----
+once to setup your workspace and before every commit
+----
+make check
+----
+
+* You *may* also run local tests on your machine or in your own development
+environment to verify everything works as expected.
+
+* The git commit messages *should* follow the rules stated on
+http://chris.beams.io/posts/git-commit/[How to Write a Git Commit Message]
```

Comment:
Feel free to give your feedback on this, e.g. the rule to capitalize each commits subject line. My personal opinion is that we should stick to the more common way of doing it, i.e. just stick to the described rules.